internal/core/autocomplete_test.go

```diff
@@ -135,8 +140,10 @@ func TestAutocomplete(t *testing.T) {
 	t.Run("scw test flower delete with-leaves=true ", run(&testCase{Suggestions: AutocompleteSuggestions{"anemone", "hibiscus"}})) // invalid notation
 	t.Run("scw test flower delete hibiscus n", run(&testCase{Suggestions: nil}))
 	t.Run("scw test flower delete hibiscus w", run(&testCase{Suggestions: AutocompleteSuggestions{"with-leaves="}}))
-	t.Run("scw test flower delete hibiscus with-leaves=true", run(&testCase{Suggestions: nil}))
+	t.Run("scw test flower delete hibiscus with-leaves=true", run(&testCase{Suggestions: AutocompleteSuggestions{"with-leaves=true"}}))
```

Comment:
this should still be nil to be iso with previous test on line 136, 129, 120,...

Reply:
Other tests you list are using `WordToCompleteIndex` this is not the case for this one

Reply:
True! We lack of a test similar to this one for enums:
```
t.Run("scw test flower create leaves.0.size=L", run(&testCase{Suggestions: AutocompleteSuggestions{"leaves.0.size=L"}}))
```